development/bootstrap-job-preparation/bootstrap-job.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
How this file should be used?

Reply:
yeh good question, can we for example do `read -s v_creds_psw` when third parameter not supplied, so it doesn't appear in history?

Reply:
It is now interactive. thanks for pointing it out